doc/examples/transform/plot_register_align.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from scipy import ndimage as ndi
+from matplotlib import pyplot as plt
+
+from skimage.data import camera
+from skimage.transform import register_affine
+from skimage import measure
+
+callback_arr = []
+callback = lambda i, m: callback_arr.append((i,m))
+r = 0.12
+out = np.empty((3, 3))
+
+out[0][0] = np.cos(r)
```

Comment:
Maybe write the whole array in a single line, rather than over 9 or 10 lines.

```
matrix_transform = [
    [np.cos(r), -np.sin(r), 0.2],
    [np.sin(r),  np.cos(r), 0.1],
    [0,          0,         1.0]]
```